FrontEnd/src/components/basics/Fallback/Fallback.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FunctionComponent } from 'react';
+import { amenityToIcon, Amenity } from '../Amenities/Amenities';
+import { Icon as IconType, miscIcons } from '@icons';
+
+// Amenity needs to be any so it can be type guarded
+interface FallbackProps {
+  amenity: any;
+}
+
+// Type guard to check if key is of Amenity type
+const isAmenity = (key: any): key is Amenity => {
+  return key in amenityToIcon;
+};
+
+// Fallback Icon Component (could be a normal amenity svg too)
```

Comment:
what does `normal amenity svg` mean? And I think a more descriptive comment of the purpose of FallbackIcon would be helpful. We want as much written down as possible so that in the future, we don't forget and don't have to ask co-workers 'what does this do? why is it useful?'